src/loops/simple_programs_with_repitition/easy/q4/ReverseReverseTest.java

```diff
@@ -0,0 +1,39 @@
+package loops.simple_programs_with_repitition.easy.q4;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class ReverseReverseTest extends BaseTest {
+
+    int input = 9268236;
+
+    public Clause[] testSentence() {
+        
+        Clause[] output = new Clause[4];
+        
+        output[0] = new StringLiteral("Enter a number : ");
+        output[1] = new NewLine();
+
+        output[2] = new StringLiteral("Reversed Number: ");
+        output[3] = new IntegerLiteral(reverse(input));
+
+        return output;
+
+    }
+
+    public void runMain() {
+        provideInput("" + input);
+        ReverseReverse.main(new String[0]);
+    }
+    
+    public static int reverse(int n) {
```

Comment:
Same as my previous comments, the method seems unecessary

Reply:
I know why he put this method here. but perhaps it's better to use the range version of `IntegerLiteral` and then just check if the number at that position in the sentence is the reverse of the input